apps/www/src/contents/docs/installation/manual.mdx

```diff
@@ -398,4 +398,22 @@ export const cn = (...classLists: ClassValue[]) => twMerge(clsx(classLists));
 
 You can now start adding components to your project.
 
+```bash
```

Comment:
I think this is what you were referring to, I'm sorry I didn't notice that the manual didn't have a CLI guide either, I must have thought it was already in the manual.